pyproject.toml

```diff
@@ -17,6 +17,7 @@ max-line-length = "88"
 max-attributes = 15
 max-args= 6
 max-locals = 16
+good-names = "f"
```

Comment:
If you get more verbose with the name as my suggestion, you can probably remove this line...
```suggestion
```